src/main/java/org/mskcc/cbio/oncokb/domain/Token.java

```diff
@@ -44,6 +44,9 @@ public class Token implements Serializable {
     @Column(name = "renewable", nullable = false)
     private Boolean renewable = true;
 
+    @Column(name = "name")
+    private String name;
```

Comment:
Since we have never required the tokens to have a name in the past, we would need to give them all names. I believe it makes more sense to just leave the name nullable.